woocommerce/wc-admin
# Show empty chart and summary numbers when search returns no results

Fixes #1451.

Follow-up of #1452.

Show empty chart and summary numbers when a search returns no results and avoid doing useless API calls.

### Accessibility


### Screenshots
![image](https://user-images.githubusercontent.com/3616980/52337288-8c63d000-2a07-11e9-9186-e6cbc76dd111.png)

### Detailed test instructions:
* Go to the _Products_ report.
* Make a search that will return no results.
* Verify the chart and summary numbers are empty.
* In your browser devtools _Network_ tab, verify there is no call sent to `/wc/v4/reports/products/stats`.
